docs/howtoguides/attach_index.rst

```diff
@@ -1,12 +1,24 @@
+.. _attach_index:
+
 Attach a machine to a subscription
 **********************************
 
-* :ref:`How to attach <attach>` shows you how to perform the attach process.
-* :ref:`How to simulate the attach operation <simulate-attach>` to check a
-  specific token to see what will happen if you attach to it (before
-  attaching anything).
-* :ref:`How to attach with a config file <attach-with-config>` to pass a
-  specific token to your machine using a configuration file.
+To access Ubuntu Pro services, you need to attach your machine to a
+subscription. The most commonly used way is to attach a single machine using
+``pro attach``:
+
+* :ref:`How to attach <attach>`
+
+If you have a paid subscription and want to check a specific token to see what
+will happen if you attach to it, you can simulate the attach operation (without
+attaching anything):
```

Comment:
instead of "without attaching anything" maybe we could something on the lines of "no changes will be made to your machine", something that explains the simulation better perhaps?

Reply:
Good idea - updated (let me know if it's better)